src/components/BulkEditList/BulkEditListResult/PreviewContainer/PreviewContainer.js

```diff
@@ -31,19 +33,21 @@ const PreviewContainer = () => {
 
   const isInitial = step === EDITING_STEPS.UPLOAD;
 
-  return isLoading ? (
-    <div className={css.LoaderContainer}>
-      <Loading />
-    </div>
-  ) : (
-    <Preview
-      title={title}
-      id={id}
-      capabilities={capabilities}
-      bulkDetails={bulkDetails}
-      isInitial={isInitial}
-    />
-  );
+  if (criteria === CRITERIA.IDENTIFIER) {
```

Comment:
Let's remove nested ternary operator. It can be 

`if (criteria != CRITERIA.IDENTIFIER) return <NoResultsMessage />`
and then
`return isLoading ? (....`